ParadiseSS13/Paradise
# Ant goes to optometrist: new eyewear sprites for Kidan

## What Does This PR Do
Adds sprites for all eyewear that is currently used for the Kidan species. Sprites not mentioned anywhere in the code or fluff items have been ommited.

> _Due to the new regulations between the Human-Skrell Association and the Provisional Government of Aurum, a great number of Kida have joined the workforces of Nanotrasen. Their prowess and deep knowledge of mining at hazardous enviroments makes them a great addition to the company, in particular on research stations focused on plasma mining. The Epsilon Eridani sector will benefit from these new assets._
> 
> _In order to acomodate all the new workers, ensure they perform adequately and comply with the compulsory Safety Regulations, the NT Equipment branch has developed a new line of eyewear products designed with the Kidan phisiology in mind. This will prevent workplace accidents while boosting employee morale and motivation._




## Why It's Good For The Game
Kida have always been one of the most forgotten species. Now that karma-species are not a thing, they should be brought to the same quality standards as the rest. The base human eyewear looked awful on them as their eyes are five times as big, so they required new spites the most.

This is my first spritework, so consider all these 'codersprites'. Any feedback or changes are welcome.


## Images of changes
![kidan-down-small](https://user-images.githubusercontent.com/58085266/226412172-c2a213bb-cbf2-4726-9d79-ffe7b97486dc.png)

<details><summary>Other directions</summary>

![kidan-left-small](https://user-images.githubusercontent.com/58085266/226412607-bcbbec1e-49e0-43df-9ed3-f732992f622d.png)
![kidan-right-small](https://user-images.githubusercontent.com/58085266/226412655-68291fdd-09c6-4123-8ec5-fd1484dabc71.png)
![kidan-up-small](https://user-images.githubusercontent.com/58085266/226412886-ea65996d-1fd0-4509-8cb0-5f104e4d4a67.png)

</details>


## Testing
- Start server
- Reenact the movie Antz (1998)
- Check that the sprites look fine in all directions
-  🐜 


## Changelog
:cl:
imageadd: Kidan have new sprites for all eyewear, fitting their shapes better.
/:cl:
